fw/devices/iidx/iidx_combo.cpp

```diff
@@ -0,0 +1,75 @@
+#include "iidx_combo.h"
+#include "iidx_usb.h"
+
+namespace Iidx {
```

Comment:
`Iidx` hurts my brain to read, thoughts on using all caps?

Reply:
Fair enough, changed.